Update Alps, remove Todi (#233)
C2SM/c2sm.github.io: docs/posts/2024-12-17_alps_update.md

---
date:
  created: 2024-12-17
categories:
  - Alps
---

# Update on the current status on Alps

> Last week, CSCS deployed the Climate and Weather vCluster `santis`. Since there is still some fine-tuning to be performed by CSCS, the [Santis section](../alps/vclusters.md#santis) provides an overview how to transition from `todi` to `santis`.

Reviewer: ```suggestion 
Last week, CSCS deployed the Climate and Weather vCluster `santis`. As some fine-tuning is still ongoing, the [Santis section](../alps/vclusters.md#santis) provides an overview of how to transition from `todi` to `santis`.
```
